react-context/flow.txt

```diff
@@ -0,0 +1,56 @@
+* User can sign up
```

Comment:
Great work on this! Looks like you have a solid understanding on the flow of data throughout the app from client to server and back.

Reply:
thanks scott!